fili-wikipedia-example/src/main/java/com/yahoo/wiki/webservice/data/config/names/WikiApiDimensionName.java

```diff
@@ -2,32 +2,43 @@
 // Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
 package com.yahoo.wiki.webservice.data.config.names;
 
+import com.yahoo.bard.webservice.data.config.names.DimensionName;
 import com.yahoo.bard.webservice.util.EnumUtils;
 
 /**
  * Hold all the Wikipedia API dimension names.
  */
-public enum WikiApiDimensionName {
-    PAGE,
-    LANGUAGE,
-    USER,
-    UNPATROLLED,
-    NEW_PAGE,
-    ROBOT,
-    ANONYMOUS,
-    NAMESPACE,
-    CONTINENT,
-    COUNTRY,
-    REGION,
-    CITY;
+public enum WikiApiDimensionName implements DimensionName {
+    PAGE("Description of Page", "wikipedia page", "General"),
+    LANGUAGE("Description of Language", "wikipedia language", "General"),
+    USER("Description of User", "wikipedia user", "General"),
+    UNPATROLLED("Description of Unpatrolled", "wikipedia unpatrolled", "General"),
+    NEW_PAGE("Description of New page", "wikipedia newPage", "General"),
+    ROBOT("Description of Robot", "wikipedia robot", "General"),
+    ANONYMOUS("Description of Anonymous", "wikipedia anonymous", "General"),
+    NAMESPACE("Description of Namespace", "wikipedia namespace", "General"),
+    CONTINENT("Description of Continent", "wikipedia continent", "General"),
+    COUNTRY("Description of Country", "wikipedia country", "General"),
+    REGION("Description of Region", "wikipedia region", "General"),
+    CITY("Description of City", "wikipedia city", "General");
```

Comment:
Since this is an example for others to look at for how to set up a good Fili instance, it would be great to show something more than default-style placeholder text for these descriptions, long names, etc.